src/AccessibilityInsights.SharedUx/Controls/PropertyInfoControl.xaml

```diff
@@ -29,7 +29,7 @@
                 </Grid.ColumnDefinitions>
                 <Border Background="#14000000" x:Name="dpFilter" Grid.Row="0" HorizontalAlignment="Stretch" BorderBrush="LightGray" BorderThickness="0" Height="24" Margin="0,0,0,5">
                     <DockPanel>
-                        <controls:FabricIconControl GlyphName="Search" GlyphSize="Custom" FontSize="18" Margin="5,0,5,0" VerticalAlignment="Center" HorizontalAlignment="Center" Foreground="#333333"/>
+                        <controls:FabricIconControl GlyphName="Search" GlyphSize="Custom" FontSize="18" Margin="5,0,5,0" VerticalAlignment="Center" HorizontalAlignment="Center" Foreground="#333333" ShowInControlView="False"/>
```

Comment:
![search icon in property info control](https://user-images.githubusercontent.com/7775527/64988764-0bb06080-d881-11e9-8e1f-8661475af6bb.png)